ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/mlag_configuration.schema.yml

```diff
@@ -23,12 +23,24 @@ keys:
       peer_address:
         type: str
         description: IPv4 Address
+      ipv4_peer_address:
+        type: str
+        description: IPv4 Address
+      ipv6_peer_address:
+        type: str
+        description: IPv6 Address
       peer_address_heartbeat:
         type: dict
         keys:
           peer_ip:
             type: str
             description: IPv4 Address
+          ipv4_peer_ip:
```

Comment:
```suggestion
          peer_ipv4_address:
```

Reply:
peer_ip and peer_address are different keys. peer_ip is used under the heartbeat sub-options.